docs/source/cpp/compute.rst

```diff
@@ -1388,6 +1388,8 @@ For timestamps inputs with non-empty timezone, localized timestamp components wi
 +--------------------+------------+-------------------+---------------+----------------------------+-------+
 | iso_calendar       | Unary      | Temporal          | Struct        |                            | \(3)  |
 +--------------------+------------+-------------------+---------------+----------------------------+-------+
+| is_leap_year       | Unary      | Temporal          | Boolean       |                            |       |
```

Comment:
Maybe be explicit and list `Timestamp, Date`?

Reply:
Indeed! Fixed.